TINY-9148: Add active element blurring before closing a form (#8336)
tinymce/tinymce: modules/tinymce/CHANGELOG.md

- Checkmark did not show in menu colorswatches. #TINY-9395
- Toolbar split buttons in advlist plugin to show the correct state when the cursor is in a checklist. #TINY-5167
- Dragging transparent elements into transparent blocks elements could produce invalid nesting of transparents. #TINY-9231

> - Closing the dialog does not scroll to the sink in Safari anymore. #TINY-9148

Reviewer: I think the changelog items should be written how they where before not how they are now at least that is the style I've seen lately in the changelog if you compare to the other items. Also is this specific to the search dialog? And I think the sink is a bit to internal for anyone external to understand so maybe better to describe what the bug looks like for the user.

```suggestion
- Closing the search/replace dialog would scroll down the document in Safari. #TINY-9148
```

Or if it's generic for all dialogs:

```suggestion
- Closing a dialog would scroll down the document in Safari. #TINY-9148
```

Or something like that.
